frontend/Dockerfile

```
@@ -3,5 +3,6 @@ WORKDIR /usr/src/app/frontend
 COPY package*.json ./
 RUN npm install
```

```suggestion
RUN npm ci
```

`npm ci` is guaranteed to not update the lock file.